pkg/scope/cluster.go

```diff
@@ -19,18 +19,27 @@ package scope
 
 import (
 	"context"
+	"crypto/tls"
+	"net/http"
+
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
 
 	"github.com/go-logr/logr"
+	"github.com/luthermonson/go-proxmox"
 	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/utils/ptr"
 	clusterv1 "sigs.k8s.io/cluster-api/api/v1beta1"
+	clustererrors "sigs.k8s.io/cluster-api/errors"
 	"sigs.k8s.io/cluster-api/util/conditions"
 	"sigs.k8s.io/cluster-api/util/patch"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/log"
 
 	infrav1alpha1 "github.com/ionos-cloud/cluster-api-provider-proxmox/api/v1alpha1"
 	"github.com/ionos-cloud/cluster-api-provider-proxmox/pkg/kubernetes/ipam"
-	"github.com/ionos-cloud/cluster-api-provider-proxmox/pkg/proxmox"
+	capmox "github.com/ionos-cloud/cluster-api-provider-proxmox/pkg/proxmox"
```

Comment:
is it necessary to override the name here?

Reply:
yes, cause we have 3 proxmox pkgs :D

Reply:
sure but none of them are named `proxmox`.